compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/TileAttention.cpp

```diff
@@ -161,6 +161,7 @@ getTileAttentionIndexingMaps(RewriterBase &rewriter, int64_t tiledInputRank,
       AffineMap::get(/*dimCount=*/4, /*symbolCount=*/0, {k2, k1}, ctx);
   AffineMap vMap =
       AffineMap::get(/*dimCount=*/4, /*symbolCount=*/0, {k2, n}, ctx);
+  AffineMap sMap = AffineMap::get(/*dimCount=*/4, /*symbolCount=*/0, {}, ctx);
```

Comment:
nit: you don't need the `{}`.

```suggestion
  AffineMap sMap = AffineMap::get(/*dimCount=*/4, /*symbolCount=*/0, ctx);
```

Reply:
Implemented